src/components/menu/Menu.tsx

```diff
@@ -204,6 +204,19 @@ const Menu = <S,>({
     }
   }, [active, handleDocumentClick, position]);
 
+  useEffect(() => {
+    const menuNode = menuRef.current;
+    if (!menuNode) {
+      return;
+    }
+
+    menuNode.addEventListener('focusout', handleDocumentClick);
```

Comment:
Q: should this also only happen for static position like the `click` event handler?

Reply:
(And do you need both this one and the click event handler?)
I suppose this will close the menu if you click on a disabled menu item too which might not be desired

Reply:
@lorgan3 I removed the event listener. The problem before was not the event listener on the document, it was the callback.